conf/telemetry/default_rotorcraft.xml

```diff
@@ -36,6 +36,7 @@
       <message name="LIDAR"                    period="1.2"/>
       <message name="INS_EKF2"                 period=".25"/>
       <message name="WIND_INFO_RET"            period="1."/>
+      <message name="AHRS_REF_QUAT"            period="0.1"/>
```

Comment:
is it really necessary to have this in the default mode as well ?

Reply:
It can be very useful for detecting attitude control/estimation problems or when investigating a crash. If you think it is not worth it we can keep it out, but we tend to add it for every AC...

Reply:
Then I would make it slower. It is even faster than the position and attitude message at the moment. You could still have a fast message in the flight recorder section.